pg_chameleon/lib/sql_util.py

```diff
@@ -274,9 +276,15 @@ def parse_create_table(self, sql_create, table_name):
 			:rtype: dictionary
 		"""
 		
+		table_dic = {}
+		
+		like_match = self.m_like.search(sql_create)
+		if like_match is not None:
```

Comment:
why the `not None` in the check? 
using `if like_match:` should do the same and in a clearer way